netbox_netprod_importer/importer.py

```diff
@@ -172,9 +172,7 @@ def get_interfaces(self):
                     napalm_ifprops["description"] or ""
                 )[:100],
                 "mac_address": napalm_ifprops["mac_address"] or None,
-                # wait for this pull request
-                # https://github.com/napalm-automation/napalm/pull/531
-                "mtu": napalm_ifprops.get("mtu", None),
+                "mtu": napalm_ifprops["mtu"] if napalm_ifprops["mtu"] else None,
```

Comment:
Change it to `napalm_ifprops.get("mtu") or None` so it handles the case where MTU is not set by napalm.

Reply:
Made without calling get method